docs/unified-test-documentation/dasharo-security/20R-bios-setup-password.md

```diff
@@ -1,4 +1,4 @@
-# Dasharo compatibility: BIOS Setup password
```

Comment:
@Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `UEFI Setup password`. I think, that in documentation we should use the same nomenclature

Reply:
Renamed